src/bin/utils/test.rs

```diff
@@ -134,16 +134,7 @@ pub fn run_tests(
     let native_context = NativeContext::new();
 
     // Compile the sierra program into a MLIR module.
-    let native_module = native_context
-        .compile_with_metadata(
-            &sierra_program,
-            MetadataComputationConfig {
-                function_set_costs: function_set_costs.clone(),
-                linear_ap_change_solver: true,
-                linear_gas_solver: true,
-            },
```

Comment:
Removing this won't break anything?

Reply:
it didn't break the tests and the tool keeps working from me, i think not.